src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/acrbuildext/setup.py

```diff
@@ -35,7 +35,12 @@
     author_email='logger@microsoft.com',
```

Comment:
kraterdev =)

Reply:
haha, I think this has been there for a long time :-)